plugins/woocommerce/src/Internal/WCCom/TrackingController.php

```diff
@@ -0,0 +1,101 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Internal\WCCom;
+
+use Automattic\Jetpack\Constants;
+use Automattic\WooCommerce\Internal\Features\FeaturesController;
+use Automattic\WooCommerce\Internal\RegisterHooksInterface;
+use Automattic\WooCommerce\Internal\Traits\ScriptDebug;
+
+/**
+ * Class TrackingController
+ *
+ * @since x.x.x
+ *
+ * phpcs:disable Generic.Commenting.DocComment.MissingShort
```

Comment:
A comment explaining the rationale behind disabling this rule in this file would be good.